App/Files/FilesUtility/readAndWriteToFiles.js

```diff
@@ -1,13 +1,36 @@
+const fs = require("fs");
+
+const content = "some content!";
+
+const path = "./Output.txt";
+
+//lets see how to creat a function that will write the file
 function* writeToTextFile() {
-    fs.writeFile('Output.txt', data, (err) => {
+    fs.writeFile('./Output.txt', content, (err) => {
+        // if an error occurs 
+        if (err) {
+
+            console.log(err);
+
+        }
+        else {
+            // if no error
+            console.log("file written successfully");
+        }
 
-        // In case of a error throw err. 
-        if (err) throw err;
     })
 }
 
 function readFromTextFile() {
+    // read from file
+    fs.readFile('./Output.txt', function read(err, data) {
+        if (err) {
+            throw err;
+        }
+        content = data;
+    });
 
 }
 
-module.exports = { writeToTextFile, readFromTextFile }
\ No newline at end of file
+writeToTextFile(content, path);
```

Comment:
```suggestion
//writeToTextFile(content, path);
```

Reply:
We shall execute the function in other files not here. just comment out the function execution